packages/utils/src/TestingUtils/JestUtils/TestWrapper.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import { MemoryRouter, Route, Routes } from 'react-router-dom';
+import { IntlProvider } from 'react-intl';
+
+const TestWrapper = ({ renderOptions = {}, children }) => {
+  const mockStore = configureStore();
+
+  return (
+    <IntlProvider locale="en">
+      <Provider store={renderOptions?.store || mockStore()}>
+        <MemoryRouter initialEntries={renderOptions?.initialEntries || ['/']}>
+          {renderOptions?.componentPath ? (
+            <Routes>
+              <Route path={renderOptions?.elementPath || '/'}>{children}</Route>
```

Comment:
Shouldn't it be using `componentPath` instead? I would also restrain from giving any route (even /) if it's not explicitly given, just render the component without route wrapping:

```suggestion
              <Route path={renderOptions.componentPath}>{children}</Route>
```